### ISSUE TITLE
Add short-cut keys list to a menu location.

### ISSUE THREAD

**Issue by mikobuntu** [LMMS/lmms#1811]
As the short-cut key combos will be growing in future development, I think it would be a good idea to have quick access to these combinations from a user perspective.

I know this info is available online , but as workflow is a high priority in LMMS it would make sense not to have the user waste valuable creation time searching online. ( the user may not have internet access either ).

I propose the "Help" menu, which will have a tabbed window with relevant grouping of catagories, much like the "About" window.

**Comment by Umcaruje** [comment]
+1

**Comment by tresf** [comment]
How about we take it a step further and make it an [enumeration](http://qt-project.org/doc/qt-4.8/qml-enumeration.html) that drives our shortcut design... 

Then we can create a table on-the-fly with the enumerated shortcut types without hard-coding the dialog with shortcuts that will change.

If we treat it as a Constant class, we can replace all instances of shortcuts with our own internal enumerated types so that we have a centralized place and we wouldn't need to bug Stian as much... :)

**Comment by mikobuntu** [comment]
@tresf  I think I understand your proposal  , remember I am only a novice wanna be coder ;) , but as always planning and making things centralized is always a good thing.

**Comment by Spekular** [comment]
Sounds great to me! If it were possible to prompt the display of a list of
shortcuts related to the current window/context that would be cool too.

**Comment by mikobuntu** [comment]
@Spekular 

          "Sounds great to me! If it were possible to prompt the display of a list of
shortcuts related to the current window/context that would be cool too."

excellent idea, perhaps the menu bar of the active window can have a button to allow access of the related shortcuts, or at least open the shortcut window at the appropriate tab?

**Comment by tresf** [comment]
> excellent idea, perhaps the menu bar of the active window can have a button to allow access of the related shortcuts, or at least open the shortcut window at the appropriate tab?

Perhaps, (nice to have) but that can get complicated to implement and maintain.  We'll see how robust this becomes. :+1:

**Comment by mikobuntu** [comment]
@tres yeah true , that is the thing... when more functionality ( sometimes unneeded ) is added, then maintaining will also become more complex, then also bloat becomes an issue.
 I'm happy with the basic needs of just having access to the shortcut list menu tho :+1:
